test/contquadratic.jl

```diff
@@ -0,0 +1,331 @@
+# Continuous quadratic problems
+
+function contquadratictest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64))
+    @testset "Testing quadratic interface with $solver" begin
+        @testset "QP1 - Quadratic objective" begin
+            # simple quadratic objective
+            # Min x^2 + xy + y^2 + yz + z^2
+            # st  x + 2y + 3z >= 4 (c1)
+            #     x +  y      >= 1 (c2)
+            #     x,y \in R
+
+            @test MOI.supportsproblem(solver, MOI.ScalarQuadraticFunction{Float64}, [(MOI.ScalarAffineFunction{Float64},MOI.GreaterThan)])
+
+            m = MOI.SolverInstance(solver)
+
+            v = MOI.addvariables!(m, 3)
+            @test MOI.getattribute(m, MOI.VariableCount()) == 3
+
+            c1 = MOI.addconstraint!(m, MOI.ScalarAffineFunction(v, [1.0,2.0,3.0], 0.0), MOI.GreaterThan(4.0))
+            @test MOI.getattribute(m, MOI.ConstraintCount()) == 1
```

Comment:
``ConstraintCount`` was renamed to ``NumberOfConstraints{F,S}``, now specific to the type of constraint